sklearn/ensemble/_gb.py

```diff
@@ -1512,7 +1512,7 @@ class GradientBoostingRegressor(RegressorMixin, BaseGradientBoosting):
 
     init : estimator or 'zero', default=None
         An estimator object that is used to compute the initial predictions.
-        ``init`` has to provide :term:`fit` and :term:`predict`. If 'zero', the
+        ``init`` has to provide :meth:`fit` and :meth:`predict`. If 'zero', the
```

Comment:
I think the original `term` is correct in both cases. Using `meth` will link back to Gradient Boosting's methods.

In this context, the `init` parameter is referring to **any** estimator that has the `fit` and `predict` methods. Linking to the glossary with `term` explains the `fit` and `predict` interfaces.

Reply:
Good point, in this case it's the other way around.